app/css/components/markdown-editor.css

```diff
@@ -116,14 +119,11 @@
             @apply ml-12;
             animation: fadeIn 0.2s forwards;
         }
-
-        & .--warning {
-            @apply flex-grow;
-            @apply text-red font-medium;
-            @apply mr-12;
-        }
     }
     & .editor-preview {
         @apply bg-backgroundColorA;
     }
+    & .scratchpad-footer {
+        @apply py-[4.5px] px-8;
```

Comment:
Computer's can't render half pixels so this generally don't what you want. In this situation, it's better to set the height on the footer (set a variable above where your comment is) then do something like `flex flex-row items-center`

Reply:
Yes, that was one of my approaches, but then deleted the variable as I didn't use it for anything.

Reply:
Computers _can_ render half pixels. It's implemented as sub-pixel rendering and work in most browsers. (But agree on the rest)

Reply:
My understanding is that most browsers (e.g. Chrome) round for paddings and margins and use sub-pixel rendering for borders, but even that is inconsistent. (e.g. this recent blog: https://medium.com/kajabi-ux/css-and-sub-pixel-rendering-the-case-of-the-clipped-border-4652c5a1b5ab)

Regardless, I'd rather stick to the range of paddings we have defined in Tailwind :)

Reply:
@dem4ron Just this one to do, then we can merge this 🙂

Reply:
ok!

Reply:
<img width="642" alt="Screenshot 2022-08-25 at 9 23 41" src="https://user-images.githubusercontent.com/66035744/186602258-ea990f83-9027-4783-b927-15574fa4b985.png">

These are the results of the new commit on my machine. Checked it on Safari, Firefox, Chrome. Looks exactly the same.